code/_onclick/other_mobs.dm

```diff
@@ -5,17 +5,9 @@
 	Otherwise pretty standard.
 */
 /mob/living/carbon/human/UnarmedAttack(atom/A, proximity)
-	if(a_intent == INTENT_HELP && src == A)
-		check_self_for_injuries()
-		return
-
 	if(!has_active_hand()) //can't attack without a hand.
 		var/obj/item/bodypart/check_arm = get_active_hand()
-		if(!check_arm)
-			to_chat(src, "<span class='notice'>You look at your arm and sigh.</span>")

```

Comment:
whoa now this is tg history

Reply:
silly goof look a few lines down, this is just combining checks